code/modules/medical/cloning.dm

```diff
@@ -60,15 +61,22 @@
 	RefreshParts()
 
 /obj/machinery/cloning/clonepod/RefreshParts()
+	var/total = 0
 	var/T = 0
 	for(var/obj/item/weapon/stock_parts/scanning_module/SM in component_parts)
 		T += SM.rating //First rank is two times more efficient, second rank is two and a half times, third is three times. For reference, there's TWO scanning modules
+		total += SM.rating
 	time_coeff = T/2
 	T = 0
 	for(var/obj/item/weapon/stock_parts/manipulator/MA in component_parts)
 		T += MA.rating //Ditto above
+		total += MA.rating
 	resource_efficiency = T/2
 	T = 0
+	if(total >= 16)
```

Comment:
for the future you could just do a ternary statement here, but it's not important enough that you have to change it

Reply:
what's a ternary statement

Reply:
you know, one of these

`upgraded = (total > 15 ? 1 : 0)`

ternary because it has 3 parts: condition, if true, if false

Reply:
sounds complicated, i think its fine as it is